src/common/utils/validation.ts

```diff
@@ -12,6 +12,8 @@ export class SemvarValidator implements StringValidator {
   }
 }
 
+// TODO:87: consolidate these with new Slug validators
```

Comment:
I'd like to get an rc out to make my API changes easier. Note there are still some small cleanup TODO's in this PR though.